### ISSUE TITLE
nvd3 with D3 JS V4

### ISSUE THREAD

**Issue by atapas** [novus/nvd3#1869]
Is there any timeline to have a release of nvd3 with D3JS Version 4.x?

**Comment by liquidpele** [comment]
No.  If you want to give it a go making it work send us a pull request!

**Comment by brmc** [comment]
Is there a list of the known difficulties holding back a v4 release?

**Comment by canda** [comment]
Duplicate of #1681

**Comment by bgth** [comment]
Can you please check #2082 and see if anyone can spare some time on the migration to v4? Thanks in advance..

**Comment by Tejeswini** [comment]
Hi 
I am using NVD3 in my angular 5 project and it supports d3 V4, Can we have support of d3 v4 in nvd3.
Please reply._Its very urgent.

**Comment by liquidpele** [comment]
Hi gents.  I've decided to make moving to d3 v4 a breaking change from the novus repo.  I have no control under the novus org, and I'd like to have more help but can't add anyone!  So, I've acquired the "nvd3" organization from another user and plan to use that for the d3 v4 break.   If you'd like to help, let me know and I'll add you to the new organization :)

https://github.com/nvd3/nvd3
